### PR TITLE
Updates to zipkin 0.17

### PR DESCRIPTION
**Pull request by adriancole**
Impacts to brave are improved codec performance and smaller brave jar!

With this update, the brave jar is down 20K to 300K

### CODE REVIEW

File: brave-mysql/src/main/java/com/github/kristofa/brave/mysql/MySQLStatementInterceptor.java [hunk 1]
```diff
@@ -68,7 +68,6 @@ public ResultSetInternalMethods postProcess(final String sql, final Statement in
     }
 
     private void beginTrace(final ClientTracer tracer, final String sql, final Connection connection) throws SQLException {
-        final String schema = connection.getSchema();
```
Comment (adriancole) [inline brave-mysql/src/main/java/com/github/kristofa/brave/mysql/MySQLStatementInterceptor.java]: unrelated cleanup

**Comment by kristofa** [comment]
👍
